src/Components/bookHub/BookCard.js

```diff
@@ -34,47 +31,60 @@ const useStyles = makeStyles({
 });
 
 export default function BookCard({
-  id, title, author, image,
+  id, title, author, image, inCarousel,
 }) {
   const classes = useStyles();
 
-  const [authorVar, setAuthor] = useState();
+  const [authorVar, setAuthor] = useState([]);
 
   const getAuthor = () => {
-    base('Creator').find(author, (err, record) => {
-      if (err) { console.error(err); }
-      setAuthor(record);
+    author.forEach((name) => {
+      if (name === 'MISSING AUTHOR' || name === 'Anonymous') { setAuthor('Anonymous'); return; }
```

Comment:
Rather than use Missing Author and Anonymous can you consolidate the error messages used?